js/sykepengesoknad/data/ettersending/ettersendingSelectors.js

```diff
@@ -0,0 +1,21 @@
+/* eslint arrow-body-style: ["error", "as-needed"] */
+export const ETTERSEND_SOKNAD_SENDER = 'ETTERSEND_SOKNAD_SENDER';
+export const ETTERSEND_SOKNAD_SENDT = 'ETTERSEND_SOKNAD_SENDT';
+export const ETTERSEND_SOKNAD_FEILET = 'ETTERSEND_SOKNAD_FEILET';
+
+export const selectEttersenderSoknad = state => state.ettersendingNav.sender || state.ettersendingArbeidsgiver.sender;
+
+export const selectEttersendtSoknad = state => state.ettersendingNav.sendt || state.ettersendingArbeidsgiver.sendt;
+
+export const selectEttersendSoknadFeilet = state => state.ettersendingNav.sendingFeilet || state.ettersendingArbeidsgiver.sendingFeilet;
+
+export const selectEttersendSoknadStatus = (state) => {
```

Comment:
Tror denne kunne fått en liten test, bare for å være sikker. (Og for å komme næremere 2k tester 😁)

Reply:
Done.